<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opciones_extras</class>
 <widget class="QWidget" name="opciones_extras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>466</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>466</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>271</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="button_agregardatos">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Agregar datos de mestruacion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_info">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Informacion Adicional</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_calendario">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Ver Calendario</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Volve al menu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
